es6/README.md

```diff
@@ -0,0 +1,406 @@
+# Linting
+
+Included is a default ESLint configuration, written for ESLint 3.x. It requires the additional modules:
+
+- `eslint-plugin-react`
+- `eslint-plugin-import`
+- `eslint-import-resolver-webpack`
+
+The lint configuration is the definitive source for rules; this document explains the most notable ones. `eslint <source-files> --fix` will fix most formatting issues, such as spacing. 
+
+## Overriding Lint Rules
+
+Some of the lint rules disallow uncommon but valid behaviour that is easily confused with/typoed from much more common behaviour. If you need to use the disallowed behaviour on purpose, use an explicit lint override in the source.
+
+For example, the `no-bitwise` rule disallows the bitwise operators `&` and `|` due to their unpredictable (but valid!) behaviour when substituted for the logical operators `&&` and `|`. To use bitwise operations in a situation that's approprate, do the following:
+
+```javascript
+// Mask off the bottom four bits of the octet
+/* eslint-disable no-bitwise */
+let maskedValue = rawValue & 0xf0
+/* eslint-enable no-bitwise */
+```
+
+## Removing Lint Rules
+
+These rules are extensive and a work in progress. If you find any of the rules to be a hindrance in your own work, you can disable them in the `.eslintrc.yml` file. 
+
+# Base Language
+
+## Always use curly brackets with control statements
```

Comment:
Is this specific to ES6? I feel like this may be a rule that we use for ES5. We can probably treat this README as an extension of the ES5 one. Document any new rules, or rules that override ES5 ones.